package.json

```diff
@@ -22,7 +22,19 @@
 		"watch:bundle": "rollup -c --watch"
 	},
 	"prettier": {
-		"singleQuote": true
+		"bracketSpacing": false,
+		"plugins": [
+			"prettier-plugin-svelte"
+		],
+		"singleQuote": true,
+		"overrides": [
+			{
+				"files": "*.svelte",
+				"options": {
+					"parser": "svelte"
+				}
+			}
+		]
```

Comment:
@fregante Is this `overrides` required? If so, it should be a Prettier bug.

Reply:
I don't remember, but it's in the docs https://github.com/sveltejs/prettier-plugin-svelte/tree/version-3?tab=readme-ov-file#setup

Reply:
Thanks.